grazer/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -39,62 +39,103 @@
     <![endif]-->
     
     <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
- 	<script src="<c:url value="/resources/bootstrap/js/bootstrap.min.js" />"></script>
-	<script src="<c:url value="/resources/bootstrap/js/main.js" />"></script>
+    <script src="<c:url value="/resources/bootstrap/js/bootstrap.min.js" />"></script>
+    <script src="<c:url value="/resources/bootstrap/js/main.js" />"></script>	
+    <script src="<c:url value="/resources/js/jquery-alert.js" />"></script>
   </head>
 
-  <body>
-  <div style="opacity: 0.1; position: absolute; top: 10px; left: 10px; ">
-    <img src="<c:url value="/resources/images/ep-logo.gif" />" />
-  </div>
-    <div class="container" style="padding-bottom: 150px;">
+  <body data-spy="scroll" data-offset="0" data-target="#navigation">
+    <div id="navigation" class="navbar navbar-default">
+      <div class="container-fluid">
+          <sec:authorize access="isAuthenticated()">
+            <div class="container">
+              <div class="row" style="padding-top: 15px;">
+                <div class="col-md-10">
+                  Welcome <i class="fa fa-user"></i><span style="margin-left: 5px;"><sec:authentication property="principal.username" /></span>!
+                  <form action="<c:url value='/logout' />" method='POST' class="pull-right">
+                    <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
+                    <div>
+                      <button type="submit" class="btn btn-link" style="color:#800000"><i class="fa fa-sign-out" aria-hidden="true"></i> Logout</button>
+                    </div>
+                  </form>
+                </div>
```

Comment:
Bootstrap grid system has 12 columns. The row inside a container may atmost span 12 columns. You may organize the code as: <div class="col-md-10">Welcome, .... </div> <div class="col-md-2"> Logout Form </div>.
Else remove the class = "col-md-10", so that the row spans the entire 12 columns. The benefit of dividing rows into columns, is that when the website is viewed on small screen devices e.g Mobile phones, the columns in a row may get stacked one above another to ensure that contents are displayed properly.  The size of the column assigned to a div may vary as per the layout design.
Refer: https://getbootstrap.com/docs/3.3/examples/grid/